wagtailvideos/ffmpeg.py

```diff
@@ -0,0 +1,73 @@
+from __future__ import absolute_import, print_function, unicode_literals
+
+import datetime
+import logging
+import os
+import re
+import shutil
+import subprocess
+import tempfile
+
+from django.core.files.base import ContentFile
+
+logger = logging.getLogger(__name__)
+
+try:
+    from shutil import which
+except ImportError:
+    from distutils.spawn import find_executable as which
+
+
+def DEVNULL():
+    return open(os.devnull, 'r+b')
+
+
+def installed(path=None):
+    return which('ffmpeg', path=path) is not None
+
+
+def get_duration(file_path):
+    if not installed():
+        raise RuntimeError('ffmpeg is not installed')
+
+    try:
+        # FIXME prints out extra stuff on travis, pip stderr to dev/null
```

Comment:
Not on purpose. Maybe the version of ffmpeg has changed? I'll remove the FIXME though if it is fixed.